.test-infra/metrics/sync/github/sync.py

```diff
@@ -218,16 +243,16 @@ def extractBeamReviewers(pr):
     reviewers.append(r['node']['author']['login'])
 
   ## Hi, @r1, can you .. look?
-  beamReviewerRegex = r'(Hi.*look?)'
+  beam_reviewer_regex = r'(@\w+\s.*)(?:PTAL|ptal|look)'
```

Comment:
Looking at the above comment, I see that this new regex doesn't support punctuation after the mention (i.e. `@r1, please take a look`).

I suggest replacing `\s` (whitespace) with `\W` (non-alphanumeric):

```python
beam_reviewer_regex = r'(@\w+\W.*)(?:PTAL|ptal|look)'
```